### ISSUE TITLE
adform splash-banner does not show at all

### ISSUE THREAD

**Issue by mardu** [iamnoah/writeCapture#16]
I found out an adform case that does not work correctly with writeCapture script. I have no idea why.
Here is the example page:

http://f.postimees.ee/test/

Unfortunately the banner won't be active forever. If it will be taken down I will find another test case.

**Comment by iamnoah** [comment]
It looks like you need either proxyGetElementById or writeOnGetElementById. The usage is described on the [Options](http://github.com/iamnoah/writeCapture/wiki/Options) wiki page.

**Comment by mardu** [comment]
Thank You, very much. That works perfectly. But is there any reason I shouldn't use proxyGetElementById in every case?

**Comment by iamnoah** [comment]
There could be weird side effects in some cases (although I don't recall encountering any).

**Comment by mardu** [comment]
I also found out that per call options does not work.

I made a test page here again:
http://f.postimees.ee/test/

At first i set the "jQuery.writeCapture.writeOnGetElementById = true;" and then I use per call options. The first one works with "{writeOnGetElementById:true}" and should work but the second one also works but should not because i set "{writeOnGetElementById:false}". Anyway, if I uncomment the global option, neither of them will work.
